mcs/class/System.ServiceModel.Syndication/System.ServiceModel.Syndication.dll.sources

```diff
@@ -1,40 +1,10 @@
 ../../build/common/Consts.cs
 ../../build/common/Locale.cs
 ../../build/common/MonoTODOAttribute.cs
+../../build/common/SR.cs
 Assembly/AssemblyInfo.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Atom10FeedFormatter.cs
```

Comment:
can we replace the System.ServiceModel.Web files too since they're the source for this?

Reply:
Done! 🙂